src/data/sentencepiece_vocab.cpp

```diff
@@ -306,6 +306,20 @@ class SentencePieceVocab : public IVocab {
     return spm_->GetPieceSize();
   }
 
+  size_t loadFromSerialized(const string_view& serialized) override {
+    LOG(info, "[data] Loading SentencePiece vocabulary from buffer");
+
+    spm_.reset(new sentencepiece::SentencePieceProcessor());
+    const auto status = spm_->LoadFromSerializedProto(serialized);
```

Comment:
Can you add a note in comments somewhere on how the mechanics work (I know very little protobuf). Does this expect memory referenced by `string_view` to be alive somewhere, or load a copy and keep it, during when the AlignedMemory that generates it be discarded?

Reply:
The note is added.